implementations/elixir/ockam/ockam_healthcheck/lib/healthcheck/healthcheck.ex

```diff
@@ -287,9 +286,37 @@ defmodule Ockam.Healthcheck do
     end
   end
 
-  def generate_identity() do
+  def get_identity() do
+    case :persistent_term.get(:healthcheck_identity, :none) do
+      :none ->
+        generate_and_cache_identity()
+
+      identity ->
+        ## Validate that identity is still valid
+        case Ockam.Identity.create_signature(identity, "something") do
```

Comment:
Is there a more meaningful argument we can provide than `"something"` here?

Reply:
We can use `""`, it doesn't really matter.
Ideally the identity service API should implement a function to check if this identity has a key in the vault.